distributed/client.py

```diff
@@ -3107,9 +3115,12 @@ def _graph_to_futures(
                     "submitting_task": getattr(thread_state, "key", None),
                     "fifo_timeout": fifo_timeout,
                     "actors": actors,
-                    "code": self._get_computation_code(
-                        nframes=dask.config.get(
-                            "distributed.diagnostics.computations.nframes"
+                    "code": tuple(
+                        f.code
```

Comment:
I think this will be most useful if we send the line number to the scheduler too

Reply:
Sure, I could do that. Although in this PR I was initially just going to capture the line number without making any additional changes to outside callers, thinking that perhaps @shughes-uk has some specific ideas in mind before shipping the line number around.

Reply:
Have updated to send this to the scheduler. :+1: